packages/rax/src/lazy.js

```diff
@@ -0,0 +1,53 @@
+import { LAZY_TYPE } from './constant';
+
+const Uninitialized = -1;
+const Pending = 0;
+const Resolved = 1;
+const Rejected = 2;
+
+function lazyInitializer(payload) {
+  if (payload._status === Uninitialized) {
+    const ctor = payload._result;
+    const thenable = ctor(); // Transition to the next state.
+
+    const pending = payload;
+    pending._status = Pending;
+    pending._result = thenable;
+    thenable.then((moduleObject) => {
+      if (payload._status === Pending) {
+        const defaultExport = moduleObject.default;
```

Comment:
`const export = moduleObject.default || moduleObject;` to support commonjs module?